MinimalApi.Tests.Integration/Tests/ResultEndpoints/WithRequest/ProblemEndpointTests.cs

```diff
@@ -0,0 +1,73 @@
+using MinimalApi.Features.ResultEndpoints.WithRequest;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MinimalApi.Tests.Integration.Tests.ResultEndpoints.WithRequest;
+
+[Collection("Endpoint")]
+public class ProblemEndpointTests(RadEndpointFixture f)
+{
+    [Fact]
+    public async Task When_InternalErrorRequested_ReturnsProblemDetails()
```

Comment:
InternalErrorRequested, ExternalErrorRequested, etc, is a weird way to name these tests.  Nobody is requesting an errors.  Simplify the name of the tests to be less strange.